### PR TITLE
chore: add triage permission for DTK

### PR DESCRIPTION
**Pull request by asterwyx**
Add triage permission for DTK to acquire assignment ability.

Log: add triage permission for DTK

### CODE REVIEW

File: teams.yaml [hunk 1]
```diff
@@ -413,6 +413,8 @@ teams:
       - 18202781743
       - FeiWang1119
     repositories_permissions:
+      triage:
+        - developer-center
```
Comment (BLumia) [inline teams.yaml:417]: 下面有一个 triage 了，直接加到那个里面就行，不然两个会重复。

**Review by BLumia — CHANGES_REQUESTED** [review]
yaml 语法错误，参见评注

File: teams.yaml (hunk 1 above)
Comment (asterwyx) [reply teams.yaml]: Done.

File: teams.yaml [hunk 2]
```diff
@@ -412,7 +412,7 @@ teams:
       - ComixHe
       - 18202781743
       - FeiWang1119
-    repositories_permissions:
+    repositories_permissions:
```
Comment (BLumia) [inline teams.yaml:415]:
```suggestion
    repositories_permissions:
```

**Pull request merged**